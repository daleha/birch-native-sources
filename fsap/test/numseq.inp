PSDRR1.gen
g
temp.numseq.out
6

0



