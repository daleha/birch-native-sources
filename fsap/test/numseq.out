        10         20         30         40         50         60         70 
ACACAACTAG GCAAGCAATT TCTTAGTTCT TTCTCACACT TTAGCATTAT AAAACATTAT CATCATGGGT 

        80         90        100        110        120        130        140 
GTTTTTAATG TTGAAGATGA AATCACTTCT GTTGTAGCAC CTGCTATACT CTACAAAGCT CTAGTTACAG 

       150        160        170        180        190        200        210 
ATGCTGATAA CCTTACTCCA AAGGTTATTG ATGCCATCAA AAGTATCGAA ATTGTTGAAG GAAACGGTGG 

       220        230        240        250        260        270        280 
TGCTGGAACC ATCAAAAAAC TCACTTTCGT TGAAGATGGT GAAACCAAAC ATGTGTTGCA CAAAGTGGAG 

       290        300        310        320        330        340        350 
TTAGTAGATG TTGCTAACTT GGCTTACAAC TATAGCATAG TTGGTGGTGT TGGATTTCCA GACACAGTTG 

       360        370        380        390        400        410        420 
AGAAGATCTC ATTCGAGGCT AAACTGTCTG CAGGACCAAA TGGAGGATCC ATTGCAAAGC TGAGTGTGAA 

       430        440        450        460        470        480        490 
ATACTTCACA AAAGGTGATG CTGCTCCTAG TGAAGAGCAA CTCAAGACTG ACAAAGCTAA GGGGGATGGT 

       500        510        520        530        540        550        560 
CTTTTCAAGG CTCTTGAGGG TTACTGTTTG GCTCATCCTG ATTACAACTA AACTATATAA TCATCAACAA 

       570        580        590        600        610        620        630 
GTGTGTTGTT ATGTATACTA TATAATCATC CTGTGTGCTT AATTTGGCTG CCAATGTAAT TTTCTGTTTT 

       640        650        660        670        680        690        700 
GTTTTTTTCC TTTTCTTTTG TTGTGGAACT AAAAGTGTGA GATTGTAAGT CATGTATACC TCTCCTTCAA 

       710        720        730 
TAAATTATAA TAATAATTTA TATATATCTT TTTT

